src/Microsoft.Health.Fhir.Api/Registration/FhirServerServiceCollectionExtensions.cs

```diff
@@ -64,6 +67,21 @@ public static class FhirServerServiceCollectionExtensions
             return new FhirServerBuilder(services);
         }
 
+        /// <summary>
+        /// Adds the export worker background service.
+        /// </summary>
+        /// <param name="fhirServerBuilder">The FHIR server builder.</param>
+        /// <returns>The builder.</returns>
+        public static IFhirServerBuilder AddExportWorker(
+            this IFhirServerBuilder fhirServerBuilder)
+        {
+            EnsureArg.IsNotNull(fhirServerBuilder, nameof(fhirServerBuilder));
+
+            fhirServerBuilder.Services.AddHostedService<ExportJobWorkerBackgroundService>();
```

Comment:
Should we check that Export is enabled before adding the background service?

Reply:
Added the check in the background service.